virttest/libvirt_xml/vm_xml.py

```diff
@@ -934,6 +951,40 @@ def get_disk_all(self):
             disks[dev] = node
         return disks
 
+    def get_disk_all_by_attr(self, reverse=False, **kwargs):
+        """
+        Return VM's disk from XML definition by attribute, None if not set
+        attribute or not found disks.
+        If mutiple attributes are specified, the disk will be returned only
+        when all attributes are matched.
+
+        Usage example:
+        1. get_disk_all_by_attr(device='disk')
+        2. get_disk_all_by_attr(type='file', device='disk')
+        3. get_disk_all_by_attr(device='cdrom', reverse=True)
```

Comment:
change to get_disk_all_by_attr(reverse=True, device='cdrom' ), the parameter order is not correct

Reply:
The order doesn't matter in this case.

Reply:
From function pointview, it doesn't matter. But it  easily make people assume that reverse=True is one of parameter in kargs

Reply:
ok, I'll update it, thanks.